SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -424,6 +424,9 @@ FSubobjectSchemaData GenerateSubobjectSpecificSchema(FCodeWriter& Writer, FCompo
 	FSubobjectSchemaData SubobjectData;
 	SubobjectData.ClassPath = ComponentClass->GetPathName();
 
+	// First letter of schema component must be uppercase
+	PropertyName[0] = FChar::ToUpper(PropertyName[0]);
```

Comment:
Put this in `UnrealNameToSchemaTypeName` instead.

Reply:
Also, remove unused functions like `SchemaRPCRequestType` and `SchemaRPCResponseType`.

Reply:
`UnrealNameToSchemaTypename`